soseki/reader/tokenization.py

```diff
@@ -10,38 +10,26 @@
 
 
 class ReaderTokenization:
-    def __init__(
-        self,
-        base_model_name: str,
-        include_title_in_passage: bool = False,
-        answer_normalization_type: str = "simple",
-    ) -> None:
+    def __init__(self, base_model_name: str, answer_normalization_type: str = "simple") -> None:
         self.tokenizer = AutoTokenizer.from_pretrained(base_model_name)
-
-        self.include_title_in_passage = include_title_in_passage
         self.answer_normalization_type = answer_normalization_type
 
-    def tokenize_input(self, question: str, passage_title: str, passags_text: str, **kwargs) -> BatchEncoding:
+    def tokenize_input(self, question: str, passage_title: str, passage_text: str, **kwargs) -> BatchEncoding:
         question = self._preprocess_question(question)
         passage_title = self._preprocess_passage_title(passage_title)
-        passags_text = self._preprocess_passage_text(passags_text)
+        passage_text = self._preprocess_passage_text(passage_text)
 
-        if self.include_title_in_passage:
-            text_a = question
-            text_b = passage_title + self.tokenizer.sep_token + passags_text
-        else:
-            text_a = question + self.tokenizer.sep_token + passage_title
-            text_b = passags_text
+        passage = passage_title + self.tokenizer.sep_token + passage_text
```

Comment:
The previous behavior when `self.include_title_in_passage is True` is now default.